File: 程式語言 V2答案檔/Python/205.py
compute_list = [0]*7
for i in range(10):
    n = int(input())
    if n >= 1 and n <= 6:
        compute_list[n] += 1
    else:
        compute_list[0] += 1

for i in range(1, 7):
    print('number' + str(i) + ':' + str(compute_list[i]))
print('error:' + str(compute_list[0]))

File: 程式語言 V2答案檔/PHP/205.php
<?php
    $count = new SplFixedArray(7);
    $error = 0;
    for($i = 0; $i < 10; $i++){
        $val = intval(fgets(STDIN));
        if($val >= 1 && $val <= 6){
            $count[$val] += 1;
        }else{
            $error += 1;
        }
    }
    for($i = 1; $i <= 6; $i++){
        printf("number%d:%d\n", $i, $count[$i]);
    }
    printf("error:%d\n", $error);
?>